apps/ledger-live-mobile/src/components/RequiresCameraPermissions/Fallback.tsx

```diff
@@ -27,6 +26,9 @@ const FallbackCameraBody: React.FC<Props> = ({
 }: Props) => {
   const { colors } = useTheme();
 
+  // benefit from component scope colors variable for IconSettings
```

Comment:
I understand why you added the comment but I'm not convinced that we need it for this.

Reply:
so i delete the comment here?